handsontable/types/settings.d.ts

```diff
@@ -144,6 +144,7 @@ export interface GridSettings extends Events {
   hiddenColumns?: HiddenColumnsSettings;
   hiddenRows?: HiddenRowsSettings;
   invalidCellClassName?: string;
+  imeFastEdit?: boolean;
```

Comment:
There is a need to add the new option to the Angular wrapper https://github.com/handsontable/handsontable/blob/develop/wrappers/angular/projects/hot-table/src/lib/hot-table.component.ts#L35.